testing/.devcontainer/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM golang:1.18.10
```

```suggestion
FROM golang:1.19.7
```

Matches the Go version we use on main https://github.com/elastic/elastic-agent/blob/96f3fa136c787acbb70e0d595704037609722b59/.go-version#LL1C7